experimental/templates/multiplatform-template/androidApp/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+plugins {
+    kotlin("multiplatform")
+    id("com.android.application")
+    id("org.jetbrains.compose")
+}
+
+kotlin {
+    android()
+    sourceSets {
+        val androidMain by getting {
+            dependencies {
+                implementation(project(":shared"))
+                implementation("androidx.appcompat:appcompat:1.5.1")
+                implementation("androidx.activity:activity-compose:1.6.1")
+            }
+        }
+    }
+}
+
+android {
+    compileSdk = 33
+    defaultConfig {
+        applicationId = "com.myapplication.MyApplication"
+        minSdk = 24
```

Comment:
We have `minSdk = 26` in the old template.

Reply:
I arranged this with our current samples that were arranged with what KMM plugin wizard produces for a KMM project

Reply:
changed to 26